Tests/ObjectMapperTests/MappableExtensionsTests.swift

```diff
@@ -44,12 +44,13 @@ struct TestMappable: Mappable, Equatable, Hashable {
 	mutating func mapping(map: Map) {
 		value <- map["value"]
 	}
-	
-	var hashValue: Int {
+
+
+	func hash(into hasher: inout Hasher) {
 		if let value = value {
-			return value.hashValue
+			hasher.combine(value)
 		}
-		return NSIntegerMax
+		hasher.combine(NSIntegerMax)
```

Comment:
This will not have the same effect as the previous code. Although is not so important.
Use an `else` statement to avoid running both `hasher.combine(_:)` lines

Reply:
Sorry for late response, I've done. PTAL @gcharita :shipit: